packages/govuk-frontend/src/govuk/common/index.mjs

```diff
@@ -133,3 +133,17 @@ export function extractConfigByNamespace(configObject, namespace) {
   }
   return newObject
 }
+
+/**
+ * Checks if GOV.UK Frontend is supported on this page
+ *
+ * Some browsers will load and run our JavaScript but GOV.UK Frontend
+ * won't be supported.
+ *
+ * @internal
+ * @param {HTMLElement} [$scope] - The `<body>` element of the document to check for support
+ * @returns {boolean} Whether GOV.UK Frontend is supported on this page
+ */
+export function isSupported($scope = document.body) {
```

Comment:
Do we have browser support for parameter defaults without Babel polyfills?

Reply:
Yup, Babel leaves that line as is in `dist`. Support looks [a bit older than what we're configured to support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#browser_compatibility). And if transpilation happened, there's [no polyfills involved](https://babeljs.io/repl#?browsers=IE%2011&build=&builtIns=false&corejs=false&spec=false&loose=true&code_lz=GYVwdgxgLglg9mABDAzgZRAB03ATlAUwBMAKAEhQjkwMQF5Ei4IQBbAsKAOgCM4iAngEpEAbwBQiRLgJQQuJBSo0uEADYBDFCgAyqblU4aYYFCQDkAczgA3EAGsAtMFwJCYIo5RYc-YuaEAbnEAXyA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=true&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Cstage-2%2Cstage-3&prettier=false&targets=&version=7.22.10&externalPlugins=&assumptions=%7B%7D). So all good there 🎉